helm_deploy/values-dev.yaml

```diff
@@ -13,6 +13,9 @@ cronjobs:
   hard_stop_licences_review_overdue:
     enabled: true
     schedule: "0 8 * * 1-5" # 8am UTC Monday - Friday
+  notify_attention_needed_licences:
```

Comment:
if you put it in values.yml, all the other services will inherit it too. 

helm merges the `values-env.yml` file on top of the `values.yml`